addon/components/connect.js

```diff
@@ -38,7 +38,7 @@ export default (stateToComputed, dispatchToActions=() => ({})) => {
         }
 
         this.actions = Object.assign({},
-          this.actions, dispatchToActions(redux.dispatch.bind(redux))
+          this.actions, dispatchToActions.call(this, redux.dispatch.bind(redux))
```

Comment:
👍 this is particularly useful for the sad souls stuck in ember <1.11 land; `(action something)` isn't a helper before the 1.13 so very often we have to depend on having `this` to get variables out. Hooray